docs/reference/commandline/create.md

```diff
@@ -145,6 +146,13 @@ then be used from the subsequent container:
     drwx--S---  2 1000 staff  460 Dec  5 00:51 .ssh
     drwxr-xr-x 32 1000 staff 1140 Dec  5 04:01 docker
 
+Set storage driver options per container. 
+
+    $ docker create -it --storage-opt size=120G fedora /bin/bash
+
+This (size) will allow to set the container rootfs size to 120G at creation time. 
```

Comment:
Note that this is no longer devicemapper specific. Windows will have support when https://github.com/docker/docker/pull/23391 is merged. ZFS also supports this since https://github.com/docker/docker/pull/21946. There might be more that I missed as well.